src/Products/PloneMeeting/MeetingItem.py

```
@@ -8449,19 +8452,22 @@ def is_decided(self, cfg, item_state=None, positive_only=False):
     def may_view_follow_up(self,
                            field_name='neededFollowUp',
                            label_ids=('needed-follow-up', 'provided-follow-up'),
+                           restricted=False,
                            suffixes=[]):
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)